src/pods/movies/reducer/filter.ts

```diff
@@ -1,3 +1,33 @@
-import { createEmptyMovieFilter } from "../viewModel";
+import { actionsEnums } from "../../../common-app/actions";
+import {MovieFilter} from './model';
 
-export const filterReducer = (state = createEmptyMovieFilter(), action) => { return state};
\ No newline at end of file
+
+type FilterState = MovieFilter;
+
+const createDefaultState = (): FilterState => ({
+    title: '',
+    genere: '',
+    year: null,
+});
+
+export const filterReducer = (state = createDefaultState(), action) => { 
+    
+    switch (action.type) {
+        case actionsEnums.FILTER_REQUEST_COMPLETED:
```

Comment:
I think this filter is not sync, let's name it

FILTER_FULL_UPDATE

Berta's case can be something like FILTER_SINGLE_FIELD_UPDATE